launch_test.go

```diff
@@ -24,6 +24,7 @@ type FakePipelineDef screwdriver.PipelineDef
 type MockRepo struct {
 	checkout func() error
 	mergePR  func(string, string) error
+	getPath  func() string
```

Comment:
For just the MockRepo or for all Repos?
In the latter case we may run into an issue where we have "Path" defined as both a field and a method.